app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -155,22 +171,20 @@ private fun threeDotButton() = onView(allOf(ViewMatchers.withId(R.id.menuButton)
 // First Run elements
 private fun assertWelcomeHeader() = onView(CoreMatchers.allOf(ViewMatchers.withText("Welcome to Firefox Preview!")))
     .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
-private fun assertAlreadyHaveAnAccountHeader() {
-    onView(CoreMatchers.allOf(ViewMatchers.withText("Already have an account?")))
-        .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
-}
 private fun assertGetTheMostHeader() {
     mDevice.wait(Until.findObject(By.res("Get the most out of Firefox Preview")), waitingTime)
 }
 private fun assertAccountsSignInButton() = onView(ViewMatchers.withResourceName("turn_on_sync_button"))
     .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
 private fun assertGetToKnowHeader() = onView(CoreMatchers.allOf(ViewMatchers.withText("Get to know Firefox Preview")))
     .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
-private fun assertChooseThemeHeader() = onView(CoreMatchers.allOf(ViewMatchers.withText("Choose your theme")))
-    .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
-private fun assertChooseThemeText() {
-    mDevice.wait(Until.findObject(By.res("Try dark theme: easier on your battery and eyes")), waitingTime)
+private fun assertChooseThemeHeader() {
+    onView(CoreMatchers.allOf(ViewMatchers.withText("Choose your theme")))
+        .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
 }
+private fun assertChooseThemeText() =
+    mDevice.wait(Until.findObject(By.res("Try dark theme: easier on your battery and eyes")), waitingTime)
```

Comment:
Actually, could we refer to the text using R.string.* ? then we probably won't have to fix the test when the text is changed.  Same for "Choose your theme"

Reply:
I spoke with @rpappalax about this yesterday. He said it's good to have tests break if the string changes so we can ensure it's intentional. Are you okay with keeping it as is?